packages/endo/bin/endo

```diff
@@ -0,0 +1 @@
+endo.js
```

Comment:
does.. this work? Is there some weird pseudo-symlink data format going on here that Node.js recognizes in a `bin:` property? I would have expected package.json to point at `endo.js`.

Reply:
I’ll point this to `endo.js` since you’re right that it’s more likely to work depending on what tool you use to install CLI’s. But, Node.js does pierce symlinks such that relative imports are always relative to the canonical location.